sig-federation/ONCALL.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+We have an oncall rotation in the SIG. The role description is as follows:
+
+* Ensure that the testgrid (https://k8s-testgrid.appspot.com/sig-federation) is green. This person will be the point of contact if testgrid turns red. Will identify the problem and fix it (most common scenarios: find culprit PR and revert it or free quota by deleting leaked resources).
+Will also report most common failure scenarios and suggest improvements. Its up to the sig or individuals to prioritize and take up those tasks.
+
+Oncall playbook: https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-federation-build-cop.md
+
+# Joining the rotation
+
+Add your name at the end of the current rotation schedule.
+
+# Oncall schedule
```

Comment:
We're going to need a process to ensure that this get refreshed periodically, and that when people are unable to do their oncall duty (vacation, illness etc), there's a well-defined process for updating this.

Here's a strawman.  If you I agree, I suggest you add it to the on-call playbook, with a link from here.

1. The rotation schedule stretches one quarter into the future (so that everyone has due warning, and can plan accordingly).
2. The current oncall is responsible for checking that, and extending it as soon as we have less than 1 quarter's worth of runway (when they come oncall, they need to check who was on last, who's on next etc anyway, so submitting a short PR to extend the cycle seems easy enough).
3. If anyone is unable to do their oncall duty, they are responsible for finding someone to swap with (by sending a PR, approved by that person).  Swapping one week for another is usually relatively uncontentious.

Thoughts?

Reply:
a quarter seems too much. Will we not add or remove members till end of quarter?
If we do allow changing then people cant really plan.

Agree on 3rd thats what we have been doing. Will add it to the doc.